TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/EditorTests/BlockLibrary/BlockBuilderUtilities.cs

```diff
@@ -125,6 +125,35 @@ internal static Block.Builder CreateSimplePropertyBlockBuilder(ShaderContainer c
             return blockBuilder;
         }
 
+        internal static Block.Builder DefineSimpleBlock(ShaderContainer container, string blockName, ShaderType inputType, string inputName, ShaderType outputType, string outputName, string outputValue)
```

Comment:
nit: This function wasn't clear to me until I saw the use, a comment might be nice saying that it builds the input/output types from two fields.
Also maybe rename "outputValue" to "outputFieldExpression" or something.